elser-ingest-speedtest/challenges/ELSER-V1.json

```diff
@@ -95,6 +95,7 @@
         "include-in-reporting": true,
         "operation-type": "bulk",
         "bulk-size": {{bulk_size | default(5)}},
+        "corpora": "size256-bert-vocab-whole-words",
         "pipeline": "elser-v1-ingest",
         "request-timeout": 60.0,
         "ingest-percentage": {{ingest_percentage | default(1)}}
```

Comment:
```suggestion
        "ingest-percentage": {{ingest_percentage | default(100)}}
```

Reply:
I actually do think that 1% is a better default than 100%, as it takes many hours to run with the full dataset. Why do you think that 100% would be better?

Reply:
Drive by review, it looked like a typo.